host/frontend/operator/webui/src/environments/environment.ts

```diff
@@ -0,0 +1,16 @@
+// This file can be replaced during build by using the `fileReplacements` array.
```

Comment:
It's not clear to me whether building the debian package without changing anything will use the prod environment or not. The default should be to use the prod environment, with the developers changing it for development as needed. If this is not the case, please make it so.

Reply:
> I think this logic can be used for making variants between cuttlefish-user and cuttlefish-orchestration in the future. WDYT?
Both of those are production deployments. However, being able to run in a "debug mode" on demand would be beneficial in both cases. Certainly something worth looking at in the future.